.azdo/pipelines/azure-pipelines.yml

```
@@ -186,7 +186,7 @@ stages:
         WaitForContainersAfterDeploy: ${{ parameters.WaitForContainersAfterDeploy }}
 
   - stage: vnexte2eDeploy
-    dependsOn:
+    dependsOn: []
```

This `[]` needs to be removed too:
```suggestion
    dependsOn:
```